etna/analysis/forecast/plots.py

```diff
@@ -257,6 +256,8 @@ def plot_backtest(
         segment_history_df = history_df[segment]
         segment_forecast_df = forecast_df[segment]
         is_full_folds = set(segment_backtest_df.index) == set(segment_forecast_df.index)
+        single_point_forecast = len(segment_backtest_df) == 1
```

Comment:
What if it is a single point in each fold?

Reply:
Shouldn't there be points but not only the lines?

Reply:
Why do we need points for this? Everything looks fine with only lines. You can look at the examples in the comments to this PR.

Reply:
It is the case `horizon=1, n_folds=3` in a given example.